crates/host/src/wasmbus/config.rs

```diff
@@ -2,8 +2,8 @@ use serde::{Deserialize, Serialize};
 use url::Url;
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
-/// Wasmbus lattice configuration
-pub struct Lattice {
+/// wasmCloud Host configuration
```

Comment:
This should still mention wasmbus, since it's the wasmbus lattice protocol, that is implemented by this host